py-polars/polars/expr/expr.py

```diff
@@ -5178,9 +5197,12 @@ def rolling_std(
             set the column that will be used to determine the windows. This column must
             be of dtype Datetime.
         closed : {'left', 'right', 'both', 'none'}
-            Define which sides of the temporal interval are closed (inclusive).
+            Define which sides of the temporal interval are closed (inclusive); only
+            applicable if `by` has been set.
         ddof
             "Delta Degrees of Freedom": The divisor for a length N window is N - ddof
+            Define which sides of the temporal interval are closed (inclusive); only
+            applicable if `by` has been set.
```

Comment:
this doesn't look right

Reply:
Good catch, was happening at another place as well.